### PR TITLE
Improve Quick Start

### PR DESCRIPTION
**Pull request by rosemarymarano**
Incorporate suggestions from Russ, make other improvements, and edit some existing docs.

### CODE REVIEW

File: docs-source/content/quickstart/create-domain.md [hunk 1]
```diff
@@ -17,58 +22,22 @@ weight: 6
...
-{{% notice note %}}
-If you set the `domainHomeImageBuildPath` property to `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt`, make sure that your `JAVA_HOME` is set to a Java JDK version 1.8 or later.
-{{% /notice %}}
+1.	Create a new image with a domain home by running the [create-domain](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script. First, copy the sample [create-domain-inputs.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml) file and update your copy with:  
+       * `domainUID`: `sample-domain1`
+       * `image`: Leave empty unless you need to tag the new image that the script builds to a different name.
+       * `namespace`: `sample-domain1-ns`
+       * `weblogicCredentialsSecretName`: `sample-domain1-weblogic-credentials`
+       * `domainHomeImageBase`: `container-registry.oracle.com/middleware/weblogic:12.2.1.3`
```
Comment (russgold) [inline docs-source/content/quickstart/create-domain.md]: Please list `weblogicCredentialsSecretName` ahead of `namespace` in order to match the order in the values file.

File: docs-source/content/quickstart/create-domain.md [hunk 2]
```diff
@@ -102,14 +71,23 @@ in the ***Security*** section.
...
+    **NOTE**: Substitute the Node IP address of the worker node for `your.server.com`. You can find it by running:
 
     ```bash
-    $ kubectl get po -n sample-domain1-ns -o wide
+    $ kubectl get pods -n sample-domain1-ns -o wide
+    ```
+    Here is an example of the output of this command:
+
+    ```
+    $ kubectl get pods -n sample-domain1-ns -o wide
```
Comment (russgold) [inline docs-source/content/quickstart/create-domain.md]: Did you find any connection between the output of this command and the address you need to supply? I did not. In my testing, I found that the address to use is simply the address of the box on which kubernetes is running. I used `localhost` on my Mac. That also works when running locally on my hosted box, and so does the host name of that box when running elsewhere (curl -v -H 'host: sample-domain1.org' busgl1108.us.oracle.com:30305/weblogic/ready). I think that the `get pods` command may have been intended for other purpose.

File: docs-source/content/samples/simple/domains/domain-home-in-image/_index.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ The script will perform the following steps:
 * Create a tag that refers to the generated Docker image.
 * Create a Kubernetes domain YAML file, `domain.yaml`, in the directory that is created above. This YAML file can be used to create the Kubernetes resource using the `kubectl create -f` or `kubectl apply -f` command.
 ```
-$ kubectl apply -f /path/to/output-directory/weblogic-domains/<domainUID>/domain.yaml
+$ kubectl apply -f /<path to output-directory>/weblogic-domains/<domainUID>/domain.yaml
 ```
 
 As a convenience, using the `-e` option, the script can optionally create the domain object, which in turn results in the creation of the corresponding WebLogic Server pods and services. This option should be used in a single node Kubernetes cluster only.
```
Comment (russgold) [inline docs-source/content/samples/simple/domains/domain-home-in-image/_index.md]: This seems to be giving options: use `-e` or not. I would suggest that Quick Start should always use it, and that the text should assume it and not give options; however, it should probably point out that it is needed.

**Comment by russgold** [comment]
Is there a way to provide a link to what the generated quick start will look like with these changes?

File: docs-source/content/quickstart/create-domain.md (hunk 2 above)
Comment (rosemarymarano) [reply docs-source/content/quickstart/create-domain.md]: I worked with Mark on it. This command displays the Node IP address and that address is supposed to be used in the CURL command to connect to the ReadyApp, `curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/ready`
Is that not the case?

File: docs-source/content/samples/simple/domains/domain-home-in-image/_index.md (hunk 3 above)
Comment (rosemarymarano) [reply docs-source/content/samples/simple/domains/domain-home-in-image/_index.md]: This is not the Quick Start document. I edited some of the others to clarify the /path/to/output-directory/ problem you pointed out to me.

Comment (rosemarymarano) [reply docs-source/content/samples/simple/domains/domain-home-in-image/_index.md]: The Quick Start does give the -e option.

File: docs-source/content/quickstart/create-domain.md (hunk 2 above)
Comment (russgold) [reply docs-source/content/quickstart/create-domain.md]: No; I found absolutely no connection between the IP addresses listed in that command's output, and what you need to use in the CURL command.

Comment (rosemarymarano) [reply docs-source/content/quickstart/create-domain.md]: I will investigate this matter further and get back to you. Thanks.

Comment (rosemarymarano) [reply docs-source/content/quickstart/create-domain.md]: I committed a change to the doc that I hope addresses your concerns satisfactorily. Please let me know if that is not the case.

Comment (russgold) [reply docs-source/content/quickstart/create-domain.md]: Did you push this new change? I don't see it.

**Comment by rosemarymarano** [comment]
Yes. It's in the third commit for the PR.On Jul 25, 2019 1:50 PM, Russell Gold <notifications@github.com> wrote:@russgold commented on this pull request.



In docs-source/content/quickstart/create-domain.md:
>  
     ```bash
-    $ kubectl get po -n sample-domain1-ns -o wide
+    $ kubectl get pods -n sample-domain1-ns -o wide
+    ```
+    Here is an example of the output of this command:
+
+    ```
+    $ kubectl get pods -n sample-domain1-ns -o wide

Did you push this new change? I don't see it.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by rjeberhard** [comment]
@russgold, the easiest way to check the generated doc is to build it yourself.  We actually document how to do that here: https://oracle.github.io/weblogic-kubernetes-operator/developerguide/documentation/

```
cd docs-source
hugo server -b http://localhost:1313/weblogic-kubernetes-operator
```

On Mac, you can "brew install hugo"

**Comment by rosemarymarano** [comment]
In particular, please review the revision to the ReadyApp curl command, shown in the last commit.

**Pull request merged**